funnel/forms/project.py

```diff
@@ -5,81 +5,147 @@
 from wtforms.ext.sqlalchemy.fields import QuerySelectMultipleField
 from wtforms.widgets import CheckboxInput, ListWidget
 
-import baseframe.forms as forms
 from baseframe import __
 from baseframe.forms.sqlalchemy import AvailableName, QuerySelectField
 from coaster.utils import sorted_timezones
+import baseframe.forms as forms
 
 from ..models import RSVP_STATUS, Project
 
 __all__ = [
-    'EventForm', 'ProjectForm', 'CfpForm', 'ProjectTransitionForm', 'RsvpForm',
-    'TicketClientForm', 'TicketTypeForm', 'ProjectBoxofficeForm', 'SavedProjectForm',
-    'ProjectScheduleTransitionForm', 'ProjectCfpTransitionForm'
+    'EventForm',
+    'ProjectForm',
+    'CfpForm',
+    'ProjectTransitionForm',
+    'RsvpForm',
+    'TicketClientForm',
+    'TicketTypeForm',
+    'ProjectBoxofficeForm',
+    'SavedProjectForm',
+    'ProjectScheduleTransitionForm',
+    'ProjectCfpTransitionForm',
```

Comment:
This list doesn't seem sorted. Did we remove sorting of imports?

Reply:
isort doesn't appear to touch `__all__`